src/org/robockets/steamworks/commands/MoveConveyorTime.java

```diff
@@ -0,0 +1,41 @@
+package org.robockets.steamworks.commands;
+
+import edu.wpi.first.wpilibj.command.Command;
+import org.robockets.commons.YAxisRelativeDirection;
+import org.robockets.steamworks.Robot;
+import org.robockets.steamworks.subsystems.Conveyor;
+
+/**
+ * @author Jake Backer
+ */
+public class MoveConveyorTime extends Command {
```

Comment:
This can be a single `MoveConveyor` command with multiple constructors depending on what you want to do.